packages/cli/src/__tests__/artificial-panic.test.ts

```diff
@@ -83,9 +83,20 @@ describe('artificial-panic formatter', () => {
       expect(e.rustStack).toBeTruthy()
       expect(e.schemaPath.replace(/\\/g, '/')) // replace due to Windows CI
         .toContain('prisma/schema.prisma')
-      expect(e).toMatchObject({
-        schema: undefined,
-      })
+      expect(e.schema).toMatchInlineSnapshot(`
```

Comment:
I know we talked about it yesterday, but could you remind me why this changed?

Reply:
Previously, `formatSchema` had two mutually exclusive input arguments, which would also appear in the `RustPanic` object: `schema` and `schemaPath`. Now, we pass both as input arguments in the form of an array of tuples.

Reply:
Got it, thank you!